packages/rlc-common/src/metadata/packageJson/buildAzureMonorepoPackage.ts

```diff
@@ -190,6 +190,7 @@ function getAzureMonorepoScripts(config: AzureMonorepoInfoConfig) {
     lint: skipLinting
       ? "echo skipped"
       : "eslint package.json api-extractor.json src test",
+    pack: `${config.shouldUsePnpmDep ? "pnpm" : "npm"} pack 2>&1`,
```

Comment:
@kazrael2119 could you help add UT here?

Reply:
added